transforms/language/pii_redactor/python/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[project]
+name = "dpk_pii_redactor_transform_python"
+version = "0.2.1.dev0"
+requires-python = ">=3.10"
+description = "PII redactor Transform for Python"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "Sowmya.L.R", email = "lrsowmya@gmail.com" },
+]
+dependencies = [
+    "data-prep-toolkit==0.2.1.dev0",
+    "presidio-analyzer>=2.2.355",
+    "presidio-anonymizer>=2.2.355",
+    "flair>=0.14.0",
+    "pandas>=2.2.2",
```

Comment:
@SowmyaLR would this work with Pandas 2.1.4 or do we really require 2.2.2 ? Not a show stopper but I did notice that google colab only supports pandas 2.1.4.. btw, THANK you for using >= . Other transforms require == and that is a nightmare... :-)